### DISCUSSION TITLE
Custom sounds discovery

### DISCUSSION THREAD

**Discussion by ZeroNotHero** [Windos/BurntToast#210] [General]
I know that in BurnToast 1.0 sounds are not supported. I also know that a Windows update broke .8.5 sounds using a path.
The only way to get sounds is by using the default Windows sounds. That's it, nothing else right?

That's 100% true. No way around that.

I cracked this last night after I started thinking the Windows way. It only uses predefined sounds. so why not just open up the registry (it took some tinkering) and added my own "Toast, Toast2, - Toast10" entries. From there i can point to a .wav or .mp3 (that's all i tested) anywhere on my PC. I modified the BurntToast.psm1 file to also include those "Toast" sounds to the list Josh provided in the psm1. 

So now when I use BurntToast and I want to use a custom sound, I simply put: -sound Toast5 (or whatever Toast#), and my "custom" sound plays when the notification pops.

Is this an ugly workaround? I wasn't sure on sharing this. I'm new to scripting, but have been coding for decades.

If there is any interest, I will post the 2 files.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
